shared/locales/permissions/permissions.en-US.properties

```diff
@@ -27,6 +27,8 @@ perm-browser=Inspect Web Pages
 perm-camera=Cameras
 perm-camera-selector-appRequest=Select a camera for {{app}}.
 perm-camera-selector-webRequest=Select a camera for {{site}}.
+perm-camera-appRequest={{app}} would like to use your camera.
```

Comment:
Furthermore, 'camera' is privileged, so at least 'app' is a must (you could have a new 'camera' app from the marketplace)